stairway/src/test/java/bio/terra/stairway/impl/MdcHelperTest.java

```diff
@@ -0,0 +1,139 @@
+package bio.terra.stairway.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.stairway.Direction;
+import bio.terra.stairway.fixtures.TestFlightContext;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.slf4j.MDC;
+
+@Tag("unit")
+class MdcHelperTest {
+  private static final Map<String, String> FOO_BAR = Map.of("foo", "bar");
+  private static final String FLIGHT_ID = "flightId" + UUID.randomUUID();
+  private static final String FLIGHT_CLASS = "flightClass" + UUID.randomUUID();
+  private static final Map<String, String> FLIGHT_MDC =
+      Map.of(MdcHelper.FLIGHT_ID_KEY, FLIGHT_ID, MdcHelper.FLIGHT_CLASS_KEY, FLIGHT_CLASS);
+  private static final int STEP_INDEX = 2;
+  private static final Direction STEP_DIRECTION = Direction.DO;
+  private static final String STEP_CLASS = "stepClass" + UUID.randomUUID();
+  private static final Map<String, String> STEP_MDC =
+      Map.of(
+          MdcHelper.FLIGHT_STEP_NUMBER_KEY,
+          Integer.toString(STEP_INDEX),
+          MdcHelper.FLIGHT_STEP_DIRECTION_KEY,
+          STEP_DIRECTION.toString(),
+          MdcHelper.FLIGHT_STEP_CLASS_KEY,
+          STEP_CLASS);
+
+  private TestFlightContext flightContext;
+
+  @BeforeEach
+  void beforeEach() {
+    MDC.clear();
+    flightContext =
+        new TestFlightContext()
+            .flightId(FLIGHT_ID)
+            .flightClassName(FLIGHT_CLASS)
+            .stepIndex(STEP_INDEX)
+            .direction(STEP_DIRECTION)
+            .stepClassName(STEP_CLASS);
+  }
+
+  static Stream<Map<String, String>> initialContext() {
+    return Stream.of(null, Map.of(), FOO_BAR);
```

Comment:
Isn't this just returning a single map? If so, I don't see the point of making the tests parameterized.